libs/hpc-ui/src/lib/theme.tsx

```diff
@@ -82,6 +82,18 @@ export const THEME = {
 } as const;
 
 export const MUI_THEME = createTheme({
+  components: {
+    MuiTextField: {
+      defaultProps: {
+        margin: 'normal',
+      },
+    },
+    MuiFormControl: {
+      defaultProps: {
+        margin: 'normal',
+      },
+    },
+  },
```

Comment:
Why are these changes made? When I run [codemods](https://mui.com/material-ui/migration/migration-v4/#run-codemods), these changes aren't made.

Reply:
Ok, thanks for your changes. Please merge them in to this branch.

The changes to margins are there because MUI has changed the default margin from "normal" to "none" on form components. CDM doesn't actually use any MUI form components so it's not a huge problem, but setting the default to what we will want for FTS Admin now made sense to me - I can move it to a separate PR if you'd like.